tests/test_notification_queries.py

```diff
@@ -0,0 +1,67 @@
+import pytest  # noqa
+from graphql_relay import to_global_id
+
+from django_ilmoitin.api.schema import NotificationTemplateNode
+from django_ilmoitin.api.schema.types import LanguageEnum
+from django_ilmoitin.utils import render_preview
+
+
+def test_query_notification_templates(graphql_client, notification_template):
+    query = """
+    {
+        notificationTemplates {
+            edges {
+                node {
+                    id
+                    preview
+                    translations {
+                        subject
+                        languageCode
+                        bodyHtml
+                        bodyText
+                    }
```

Comment:
Translation specific previews should probably be tested as well (?)

Reply:
Good catch!